modules/test/playwright/tests/portal-workflow-kaleo-designer-web/notification.spec.ts

```diff
@@ -193,3 +210,64 @@ test('create a notification using role type', async ({
+	} as WorkflowDefinition;
+
+	const workflowDefinition =
+		await apiHelpers.headlessAdminWorkflow.postWorkflowDefinitionSave(
+			workflowDefinitionName,
+			workflowDefinitionContent
+		);
+
+	workflowDefinitionIds.push(workflowDefinition.id);
+
+	await processBuilderPage.goto();
+
+	await processBuilderPage.clickWorkflowDefinitionName(
+		workflowDefinitionName
+	);
+
+	await diagramViewPage.clickNode('Task');
+
+	const notificationEntry = processBuilderPage.page.getByRole('link', {
+		name: 'notification 1',
+	});
+
+	await expect(notificationEntry).toBeVisible();
```

Comment:
Could you check if this expect is necessary? The click function already run some expecteds by default before perform a click action.

Reply:
you're right, it's not, thanks!

Reply:
it's not, thank you!